src/Protocol/Diaspora.php

```diff
@@ -1698,15 +1702,16 @@ private static function receiveConversation(array $importer, $msg, $data)
 	/**
 	 * Processes "like" messages
 	 *
-	 * @param array  $importer Array of the importer user
-	 * @param string $sender   The sender of the message
-	 * @param object $data     The message object
+	 * @param array  $importer  Array of the importer user
+	 * @param string $sender    The sender of the message
+	 * @param object $data      The message object
+	 * @param int    $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.